front/src/app/todo-list/components/active-task-group/active-task-group.component.html

```diff
@@ -1,3 +1,21 @@
-<div *ngFor="let task of taskList?.tasks" class="task">
-  {{task.title}}
+<div *ngIf="activeGroupId !== undefined" class="task-list">
+  <div *ngFor="let task of taskList?.tasks" class="task-list__task">
+    {{task.title}}
+  </div>
+  <div *ngIf="!isNewTaskInputVisible">
+    <button mat-button (click)="showNewTaskInput(newTaskInput)">
+      <mat-icon>add</mat-icon>
+      new task

```

Comment:
For all buttons type each significant word starting with uppercase letter like "New Task" or "Create a Task"